<?xml version="1.0" encoding="UTF-8"?>

<launch>

  <node pkg="eagleye_geo_pose_converter" name="geo_pose_converter_node" exec="geo_pose_converter" output="screen" >
    <!-- /eagleye/fix or /GNSS/fix(GNSS pose only mode)-->
    <remap from="eagleye/fix" to="eagleye/fix" />

    <!-- plane rectangular coordinate number -->
    <param name="plane" value="7"/>
    <!-- 1 : plane rectangular coordinate  2 : MGRS -->
    <param name="tf_num" value="1"/>
    <!-- 0 : No convert  1 : ellipsoid -> altitude  2 : altitude -> ellipsoid -->
    <param name="convert_height_num" value="0"/>
    <!-- 0 : EGM2008-1  1 : GSIGE2011 Ver2.1 -->
    <param name="geoid_type" value="0"/>

    <param name="parent_frame_id" value="map"/>
    <param name="child_frame_id" value="eagleye_base_link"/>
    <param name="base_link_frame_id" value="base_link"/>
    <param name="gnss_frame_id" value="gnss_link"/>

  </node>

</launch>
